BizOps onboarding (#2607)
sourcegraph/about: handbook/ops/bizops/index.md

# Business Operations & Strategy

The Business Operations & Strategy (BizOps) team operates with the mission of driving scalable growth through cross-functional internal projects that expedite progress towards company goals.

## Members

- [Shaan Sheth](../../../../company/team/index.md#shaan-sheth-he-him), Director of Business Operations & Strategy
    - [Eric Brody-Moore](../../../../company/team/index.md#eric-brody-moore), Business Operations Manager
    - [Farhan Attamimi](../../../../company/team/index.md#farhan-attamimi), Business Operations Analyst
    - S.H., Business Operations Analyst (starting 2021-03-08)
    - K.B., Business Operations Analyst (starting 2021-03-08)

## Functional areas

- [Data and analytics](analytics.md)

## Resources

- [Onboarding](onboarding.md)

## Goals

TODO

## How to work with us

**Internal BizOps projects:**  We use [GitHub issues in the Sourcegraph analytics repository](https://github.com/sourcegraph/analytics/issues) to document projects, and the [analytics kanban board](https://github.com/orgs/sourcegraph/projects/63) for team-wide prioritization and current projects.

> **Cross-functional projects:** Tag @sourcegraph/bizops in existing [sourcegraph/sourcegraph GitHub issues](https://github.com/sourcegraph/sourcegraph/issues) or create a new issue in the BizOps repository. For larger projects, please work with the BizOps team to lay out the following information to help us understand and prioritize your request:

Reviewer: Can you make "BizOps repository" a link here?

- **People:** Who is involved, and what are the expectations of each person? Who will be responsible for driving the project forward? Does each person have the necessary bandwidth to uphold the expecations asked of them? 
- **End state:** What is the end state of the project? What decisions will be made? Be as specific as possible (i.e. if the end state is a graph, draw out the graph). Please provide a "data roadmap" if there are going to be multiple phases of the project. 
- **The why:** How does it support [Sourcegraph's company and team goals](../../../company/goals/index.md)?
- **Timeline:** What's the timeline of the project? When does this need to be delivered, and how will it be followed up upon in the future?

**Small asks and questions:** Post in the #analytics or #business-ops channel.

> To reach us, mention @business-team in the most relevant channel (i.e. #marketing for a marketing-related question).

Reviewer: ```suggestion
To reach us, simply mention @business-team in the relevant channel or thread.
```

The wording here just felt a bit unclear.

> If you need clarification or are not sure about any of the above, ask a question in #analytics or @business-ops channel.

Reviewer: ```suggestion
If you need clarification or are not sure about any of the above, ask a question in #analytics or #business-ops channel.
```
